### PR TITLE
Nerfs the cybersun hardsuit and moves it out of the uplink

### PR DESCRIPTION
**Pull request by ghost**
:cl: qustinnus
 balance: Nerfs the cybersun hardsuit and moves it out of the uplink
/:cl:

some of the best armor against lasers in the game (even beating armor specifically described as the best against lasers ICly). Don't think it should be in uplinks, and should instead stay in the forgotten ship.

### CODE REVIEW

**Comment by itseasytosee** [comment]
I've notest that the forgotten ship has appeared literally every round and I don't think that's right.

**Comment by Qustinnus** [comment]
its on purpose, its really dumb.

**Comment by itseasytosee** [comment]
I'm gonna fix that.

**Review by Rohesie — DISMISSED** [review]

**Comment by Krysonism** [comment]
It is basically a 14 tc  "I AM VALID"-sign. I don't think it is that unfair.

**Comment by EgorDinamit** [comment]
Someone post Shotgun Copypasta.
But seriously, it's basically hardsuit version of reflector vest, though realistically, reflector vest protects from 80% of laser damage and you **can buy unlimited amount of those in cargo**.

**Comment by Qustinnus** [comment]
image
like I said in my OP, it has the best laser armor compared to even armor specifically designed for laser armor. good try at making this item not sound OP though.

**Comment by Shadowflame909** [comment]
Just make it a 20 tc hardsuit and buff it to full laser immunity

**Comment by EgorDinamit** [comment]
@Qustinnus, if hardsuit had 80% damage reduction it would be the same as reflector vest, given the chance to reflect.

**Comment by necromanceranne** [comment]
Shadowflame why would you not just buy sleeping carp at that point.

**Comment by Shadowflame909** [comment]
> 
> 
> Shadowflame why would you not just buy sleeping carp at that point.

full laser immunity, even when stunned!

File: code/modules/ruins/spaceruin_code/forgottenship.dm [hunk 1]
```diff
@@ -133,7 +133,7 @@ GLOBAL_VAR_INIT(fscpassword, generate_password())
 	icon_state = "cybersun"
 	item_state = "cybersun"
 	hardsuit_type = "cybersun"
-	armor = list("melee" = 30, "bullet" = 40, "laser" = 70, "energy" = 70, "bomb" = 30, "bio" = 100, "rad" = 60, "fire" = 60, "acid" = 60)
+	armor = list("melee" = 30, "bullet" = 40, "laser" = 55, "energy" = 55, "bomb" = 30, "bio" = 100, "rad" = 60, "fire" = 60, "acid" = 60)
```
Comment (EgorDinamit) [inline code/modules/ruins/spaceruin_code/forgottenship.dm:136]:
```suggestion
	armor = list("melee" = 30, "bullet" = 40, "laser" = 60, "energy" = 60, "bomb" = 30, "bio" = 100, "rad" = 60, "fire" = 100, "acid" = 100)
```

File: code/modules/ruins/spaceruin_code/forgottenship.dm [hunk 2]
```diff
@@ -144,7 +144,7 @@ GLOBAL_VAR_INIT(fscpassword, generate_password())
 	hardsuit_type = "cybersun"
 	name = "Cybersun hardsuit"
 	desc = "Prototype hardsuit with experimental armor plates, protecting from laser-based weapons very well, while giving limited protection against anything else. Requires the user to activate the inner mechanism in order to unequip it, making it really difficult to take it off from somebody else."
-	armor = list("melee" = 30, "bullet" = 40, "laser" = 70, "energy" = 70, "bomb" = 30, "bio" = 100, "rad" = 60, "fire" = 60, "acid" = 60)
+	armor = list("melee" = 30, "bullet" = 40, "laser" = 55, "energy" = 55, "bomb" = 30, "bio" = 100, "rad" = 60, "fire" = 60, "acid" = 60)
```
Comment (EgorDinamit) [inline code/modules/ruins/spaceruin_code/forgottenship.dm:147]:
```suggestion
	armor = list("melee" = 30, "bullet" = 40, "laser" = 60, "energy" = 60, "bomb" = 30, "bio" = 100, "rad" = 60, "fire" = 100, "acid" = 100)
```

**Review by EgorDinamit — COMMENTED** [review]
Make it just a _little bit_ worse than armour you can buy from cargo en masse.

**Comment by EgorDinamit** [comment]
On your place, I'd leave the hardsuit in Nuke Ops' uplinks, since it's just a little bit better than Elite Hardsuit on lasers, and worse on everything else.

**Comment by ExcessiveUseOfCobblestone** [comment]
the vest doesnt protect from legs nor does it matter if you can buy the reflector vest considering ops have ballistics instead of energy?

why does a role about vibing in space and sometimes sneaking on the station for loot need heavy armor anyways?

File: code/modules/ruins/spaceruin_code/forgottenship.dm (hunk 2 above)
Comment (Qustinnus) [reply code/modules/ruins/spaceruin_code/forgottenship.dm:147]: no.

File: code/modules/ruins/spaceruin_code/forgottenship.dm (hunk 1 above)
Comment (Qustinnus) [reply code/modules/ruins/spaceruin_code/forgottenship.dm:136]: no.

**Comment by optimumtact** [comment]
Egor like every single PR you have made has been pretty disastrous and mostly seems like wish fulfilment.

You need to go back to the basics and evaluate your changes with a critical eye.

**Comment by Eskjjlj** [comment]
Change the strip delay to a sane value please. This is at least 5 times as much time as stripping a blood-red hardsuit

**Comment by Pugie** [comment]
> Someone post Shotgun Copypasta.
> But seriously, it's basically a hardsuit version of reflector vest, though realistically, reflector vest protects from 80% of laser damage and you **can buy unlimited amount of those in cargo**.

Moot point, reflector helmets don't exist.

**Comment by Tlaltecuhtli** [comment]
>b-b-but the specialized item to be energy resistant as +10 laser  ! !! !
AND ISNT SPACEPROOF OR HAVE 30 MELEE 40 BULLET IN IT FOR FREE

**Comment by Chaznoodles** [comment]
>Nerfs
>And removes

Pick one

**Comment by EgorDinamit** [comment]
@Qustinnus 
The main point of this hardsuit was being shit against melee and bullets, while seriously protecting from laser weaponry.

**Comment by Qustinnus** [comment]
okay but it's OP and I'm moving it's strip delay to 5 seconds just like every other hardsuit.

**Comment by EgorDinamit** [comment]
5 is too low, removing one of the reasons you wore it, set it to 30 seconds at minimum.
If you really think it's a good idea, change the description of hardsuit too.

**Comment by Qustinnus** [comment]
it's the same as other hardsuits.

**Comment by EgorDinamit** [comment]
It's supposed to have a big strip_delay, it's not supposed to be the same as other hardsuits at all.

**Comment by Qustinnus** [comment]
why does it need to be so much better than everything else

**Comment by FloranOtten** [comment]
A high strip delay only serves to make it harder to take off. The only way this could be useful and not just annoying is combat stripping. However, given they have guns, if you're stripping the hardsuit they're probably dead and/or in cuffs.

**Comment by EgorDinamit** [comment]
High strip delay was serving as a good chance to escape from handcuffs while someone is trying to remove the hardsuit.

**Comment by Pugie** [comment]
That will never ever work.

**Comment by ExcessiveUseOfCobblestone** [comment]
> why does a role about vibing in space and sometimes sneaking on the station for loot need heavy armor anyways?

**Pull request merged**